### PR TITLE
Make the Sheet widget user-theme compliant.

### PR DESCRIPTION
**Pull request by SkyzohKey**
… values.

### CODE REVIEW

File: src/UI/MainWindow.vala [hunk 1]
```diff
@@ -155,6 +155,7 @@ namespace Spreadsheet.UI {
             scrolled.add (viewport);
 
             var sheet = new Sheet ();
+            sheet.set_size_request (2000, 2000);
```
Comment (Bat41) [inline src/UI/MainWindow.vala]: Maybe in `Sheet.draw` calculate the size depending on the number of columns and lines?

File: src/Widgets/Sheet.vala [hunk 2]
```diff
@@ -111,23 +111,20 @@ namespace Spreadsheet.Widgets {
         }
 
         public override bool draw (Context cr) {
-            RGBA select_bg = { 205.0, 232.0, 245.0, 1 };
-            RGBA select_border = { 0, 136.0, 204.0, 1 };
-            RGBA gray_bg = { 200.0, 200.0, 200.0, 1 };
-            RGBA light_gray = { 51.0, 51.0, 51.0, 1 };
+
```
Comment (Bat41) [inline src/Widgets/Sheet.vala]: Please no empty line at the beginning of a method.

File: src/Widgets/Sheet.vala [hunk 3]
```diff
@@ -111,23 +111,20 @@ namespace Spreadsheet.Widgets {
...
+            var main_window = this.get_toplevel ();
+            Gtk.StyleContext style = main_window.get_style_context ();
```
Comment (Bat41) [inline src/Widgets/Sheet.vala]: Why `Gtk`? `StyleContext` or even `var` would be better I think.

**Comment by ryonakano** [comment]
I've addressed the following changes to this PR:

* Fix the speed to render cells is slower than `master`
* Allow to set customized font/cell styles
* Cleaning the code

And now this PR looks like:

Screenshot from 2019-05-02 18-17-57

(In the light mode)

Screenshot from 2019-05-02 18-17-52

(in the dark mode)

However, I think it would be even make more sense to keep the background color of cells white even in the dark mode with the following reasons:

* When we provide a way to print the sheet or export it to PDF in the future, users may be confused because the background colors are different between the display and the paper/PDF
* It seems Microsoft Office apps like Word, Excel, and PowerPoint render the background color of documents white even in the dark mode

As a consequence, I think the app should look like:

Screenshot from 2019-05-02 18-28-26

(in the light mode)

Screenshot from 2019-05-02 18-28-32

(in the dark mode)

@BaptisteGelez @SkyzohKey What do you think?

**Comment by BaptisteGelez** [comment]
Indeed, I didn't thought about that, but it seems more reasonable to keep the cell background white even in dark mode.

**Comment by ryonakano** [comment]
Addressed!

**Pull request merged**